modules/NodeColorManager/src/main/nbm/manifest.mf

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+OpenIDE-Module-Name: Node Color Manager
+OpenIDE-Module-Short-Description: allow you share node colors among multi graph
+OpenIDE-Module-Long-Description: \
```

Comment:
@mbastian thank you very much. I've made this manifest correct by adding white space before lines, and maven has no error, however, using HTML <a> tag result like this in gephi.
![plugin panel](http://funkyimg.com/i/27w3w.png)
after you click the link, it result like this.
![result](http://funkyimg.com/i/27w3s.png)
so I would prefer not using HTML tags.